recipes/sos-matlab/meta.yaml

```diff
@@ -0,0 +1,63 @@
+{% set name = "sos-matlab" %}
+{% set version = "0.18.3" %}
+
+package:
+  name: "{{ name|lower }}"
+  version: "{{ version }}"
+
+source:
+  url: https://pypi.io/packages/source/{{ name[0] }}/{{ name }}/{{ name }}-{{ version }}.tar.gz
+  sha256: 54bf4ddcea30d43a90fc8a7432f990aae98351b9f0de4dc64196424ea1de99b9
+
+build:
+  skip: true  # [py<36]
```

Comment:
Maybe make this `noarch: python`, remove this line, and add `python >=3.6` below

Reply:
Done, although this would not help us figure out the source of dependency conflicts.